frontend/src/app/forms/forms.page.ts

```diff
@@ -138,6 +137,14 @@ export class FormsPage implements OnInit, OnDestroy {
       const nextMenuIndex = this.sideMenuList.indexOf(this.currentMenu) + 1;
       this.navigateToStep(this.sideMenuList[nextMenuIndex]);
     } else {
+      const key = (
+        this.overallInfo.value.basicInfo.firstName +
+        '-' +
+        this.overallInfo.value.basicInfo.lastName +
+        '-' +
+        Math.floor(Math.random() * 1000000).toString()
+      ).replace(/\s+/g, '_');
+      this.storage.addItem(key, this.overallInfo.value).then(() => {});
```

Comment:
Fine for now. But for the future, I guess we should improve this (as you already mentioned). I think it would be great just to use an ID (auto-increment fashion) for the items.

Also, it would be really cool if we could use more than one table. Not sure if this is easily possible in Ionic Storage. But it would help a lot if there was one table for the Job Applications, one for some general settings, ...

Reply:
Btw: I know that auto-increment is not possible in Ionic Storage. But we could maybe just use a separate storage item for that :) (so: an item that holds the current "auto increment" number)

Reply:
Ya It would be good if we have more db. It can be structured really well, If we achieve it. There is an open issue related to ionic storage in it https://localforage.github.io/localForage/#multiple-instances-createinstance . I have started checking to achieve this with localforage.

Reply:
Wondering if this works: https://stackoverflow.com/questions/55470808/ionic-3-multiple-tables-in-indexeddb

Then we could have multiple Services (one for each table)

Reply:
Will check